documentacion/como_empezar/README.md

```diff
@@ -343,6 +343,51 @@ resguardar su llave privada y su certificado público, como asimismo es
 responsable por reemplazar estos cuando caduquen.
 </aside>
 
+### Configuración para producción utilizando los SDK
+
+Para realizar la configuración del ambiente de producción utilizando los SDK oficiales
```

Comment:
is it me or it's not really clear to read, I would suggest something like this " Para realizar la configuracion del emabiente de Produccion se deben utilizar los SDK oficiales